Resources/public/js/countryCodeChange.js

```diff
@@ -1,34 +1,31 @@
-module.exports = {
-    // The address form fields are initialized here, these form fields are common for some entity forms and are therefore put in this function
-    initialize: function (entity) {
-        $(document).ready(function () {
-            // When the country code changes the form will be submitted so we get a validated form for that country
-            // Only the address part here is important to change in the current view
-            var onCountryCodeChange = function () {
-                var $form = $(this).closest('form');
-                var $address = $form.find('#' + entity + '_address');
+export function initialize (entity, fieldName = 'address') {
```

Comment:
This logic has also been reworked to no longer need the entity or fieldName at all, it will just initialize the address fields automatically by initializing. I'd suggest to look at that too.